e2e/scenario/modifiers.go

```diff
@@ -0,0 +1,11 @@
+package scenario
+
+import "strconv"
+
+func stringToInt32(s string) int32 {
+	i, err := strconv.ParseInt(s, 10, 32)
+	if err != nil {
+		panic(err)
```

Comment:
Allows me to keep the int32 conversion just like it was in the past. Since its the test code anyway (that converts values that user added to their scenario), I figured its not a big deal if we panic here - at least you'll know fast that your scenario config is incorrect.

Reply:
makes sense